draft-trammell-taps-interface.md

```diff
@@ -458,22 +461,28 @@ is to not have this option.
 
 ### Multistream Connections in Group {#prop-multistream}
 
-This boolean property specifies that the application would prefer multiple
+Type: Preference
+
+This property specifies that the application would prefer multiple
 Connections within a Connection Group to be provided by streams of a single
 underlying transport connection where possible. This is not a strict
 requirement. The default is to not have this option.
 
 ### Notification of excessive retransmissions {#prop-retrans-notify}
 
-This boolean property specifies whether an application considers it
+Type: Boolean
```

Comment:
I'm not sure how this is supposed to work. Is true == Require or Prefer; is false == Avoid or Prohibit? How do I ignore?

I think this might be a Preference, though not one that most apps will use.

Reply:
Good question – I thought of it only as boolean whether to fire an event when available these notifications are available.
If we assume there are applications that do not work without this kind of notification, we have to make this a Preference, otherwise, I think it is a Boolean.

Reply:
IMO we should ask a slightly different question here, not "can we think of an application that would really want to prohibit selection of a transport based on the inability to see RTX notifications or ICMP soft errors", but "do we have a good reason for creating a third class of selection parameter when two suffice"? If True = "prefer" and False = "ignore", then we can just say that in the docs rather than specifying additional codepaths (every codepath, remember, is another way you can accidentally screw up :) )

Reply:
I don't see a functional difference between preference with two valid levels and boolean, just, from an application programmers perspective, I still think that "Boolean" is less confusing here.

I would like to get at least a second opinion from @tfpauly on that -- after that, I change it to the one that makes this change land for -00.